include/cantera/oneD/Flow1D.h

```diff
@@ -426,7 +426,10 @@ class Flow1D : public Domain1D
     }
 
     //! Update the transport properties at grid points in the range from `j0`
-    //! to `j1`, based on solution `x`.
+    //! to `j1`, based on solution `x`. Evaluates the solution at the midpoint
```

Comment:
```suggestion
    //! to `j1`, based on solution `x`. 
    //! Evaluates the solution at the midpoint
```
Text up to first period is converted to a 'brief' description (line break is technically not required). I'd recommend succinct descriptions that fit on a single line.